setup.py

```
@@ -18,10 +39,17 @@ class BdistWheel(bdist_wheel):
     def get_tag(self):
         python, abi, platform = super().get_tag()
         if python.startswith("cp"):
-            python, abi = "cp38", "abi3"
+            python, abi = "cp310", "abi3"
         return python, abi, platform
 
 
+class EggInfo(egg_info):
+    def find_sources(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `find_sources`

Add return type annotation: `None`

(ANN201)